### PR TITLE
feat: add CustomComicId and CustomEpId

### PR DESCRIPTION
**Pull request by Initial-heart-1**
Allows user to customise comics and page numbers read by comic reading task.
No need to know which comic was read, so instead of getting it, just remove from log. It can be done again if an issue calls for it.
Allow edits by maintainers.

### CODE REVIEW

**Comment by Initial-heart-1** [comment]
Notice: I'm not going to test this pull request because I think it will just fail due to user error.

File: src/Ray.BiliBiliTool.Config/Options/DailyTaskOptions.cs [hunk 1]
```diff
@@ -77,6 +77,15 @@ public string ChargeComment
         /// </summary>
         public string DevicePlatform { get; set; } = "android";
 
+        /// <summary>
+        /// 自定义漫画阅读 comic_id
+        /// </summary>
+        public int CustomComicId { get; set; } = 27355;
```
Comment (RayWangQvQ) [inline src/Ray.BiliBiliTool.Config/Options/DailyTaskOptions.cs:83]: May be we can change int to long

File: src/Ray.BiliBiliTool.Console/appsettings.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
     "ChargeComment": "", //充电后留言
     "DayOfReceiveVipPrivilege": 1, //每月几号自动领取会员权益的[-1,31]，-1表示不指定，默认每月1号；0表示不自动领取
     "DayOfExchangeSilver2Coin": 0, //每月几号执行银瓜子兑换硬币[-1,31]，-1表示不指定，默认每月最后一天；-2表示每天；0表示不进行兑换
-    "DevicePlatform": "android" //执行客户端操作时的平台 [ios,android]
+    "DevicePlatform": "android", //执行客户端操作时的平台 [ios,android]
+    "CustomComicId": "27355", //自定义漫画阅读 comic_id，若不清楚含义请勿修改
```
Comment (RayWangQvQ) [inline src/Ray.BiliBiliTool.Console/appsettings.json:19]: Why not using int but string? I saw it is defined as int in option class.

**Review by RayWangQvQ — CHANGES_REQUESTED** [review]

File: src/Ray.BiliBiliTool.Console/appsettings.json (hunk 2 above)
Comment (Initial-heart-1) [reply src/Ray.BiliBiliTool.Console/appsettings.json:19]: Yes, you are right. Just to make sure the value was captured, I added double quotes out of habit.

File: src/Ray.BiliBiliTool.Config/Options/DailyTaskOptions.cs (hunk 1 above)
Comment (Initial-heart-1) [reply src/Ray.BiliBiliTool.Config/Options/DailyTaskOptions.cs:83]: I think that int is sufficient, but of course I can also use long if you insist on it.

**Comment by Initial-heart-1** [comment]
**WARN**
Changes were just pulled from develop branch to resolve merge conflict, but unexpectedly CodeQL reported that build failed.

File: src/Ray.BiliBiliTool.DomainService/MangaDomainService.cs [hunk 3]
```diff
@@ -62,11 +62,11 @@ public async Task MangaSign()
         /// </summary>
         public async Task MangaRead()
         {
-            BiliApiResponse response = await _mangaApi.ReadManga(_dailyTaskOptions.DevicePlatform);
+            BiliApiResponse response = await _mangaApi.ReadManga(_dailyTaskOptions.DevicePlatform, _dailyTaskOptions.CustomComicId, _dailyTaskOptions.CustomEpId);
```
Comment (RayWangQvQ) [inline src/Ray.BiliBiliTool.DomainService/MangaDomainService.cs:65]:
```suggestion
            if ( _dailyTaskOptions.CustomComicId <= 0 ) return;
            BiliApiResponse response = await _mangaApi.ReadManga(_dailyTaskOptions.DevicePlatform, _dailyTaskOptions.CustomComicId, _dailyTaskOptions.CustomEpId);
```

**Review by RayWangQvQ — APPROVED** [review]

**Comment by RayWangQvQ** [comment]
> **WARN** Changes were just pulled from develop branch to resolve merge conflict, but unexpectedly CodeQL reported that build failed.

Maybe you can try pull the latest codes from upstream, then **rebase** your feature branch to develop branch.

File: src/Ray.BiliBiliTool.Agent/BiliBiliAgent/Interfaces/IMangaApi.cs [hunk 4]
```diff
@@ -26,8 +26,8 @@ public interface IMangaApi : IBiliBiliApi
         /// </summary>
         /// <param name="platform"></param>
         /// <returns></returns>
-        [HttpPost("/twirp/bookshelf.v1.Bookshelf/AddHistory?platform={platform}&comic_id=27355&ep_id=381662")]
-        Task<BiliApiResponse> ReadManga(string platform);
+        [HttpPost("/twirp/bookshelf.v1.Bookshelf/AddHistory?platform={platform}&comic_id={comic_id}&ep_id={ep_id}")]
+        Task<BiliApiResponse> ReadManga(string platform, int comic_id, int ep_id);
```
Comment (RayWangQvQ) [inline src/Ray.BiliBiliTool.Agent/BiliBiliAgent/Interfaces/IMangaApi.cs:30]:
```suggestion
        Task<BiliApiResponse> ReadManga(string platform, long comic_id, long ep_id);
```

Comment (Initial-heart-1) [reply src/Ray.BiliBiliTool.Agent/BiliBiliAgent/Interfaces/IMangaApi.cs]: How awkward...Fixed but forgot this😂

**Pull request merged**